.github/workflows/cloud.yml

```diff
@@ -0,0 +1,57 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: "ClickHouse Cloud"
+
+on:
+  push:
+    branches:
+      - "branch-*"
+      - "main"
+  pull_request:
+    branches:
+      - "branch-*"
+      - "main"
+
+jobs:
+  run-tests-with-clickhouse-cloud:
+    runs-on: ubuntu-22.04
+    strategy:
+      fail-fast: false
+      matrix:
+        spark: [ 3.3, 3.4, 3.5 ]
+        scala: [ 2.12, 2.13 ]
+    env:
+      CLICKHOUSE_CLOUD_HOST: ${{ secrets.INTEGRATIONS_TEAM_TESTS_CLOUD_HOST_SMT }}
+      CLICKHOUSE_CLOUD_PASSWORD: ${{ secrets.INTEGRATIONS_TEAM_TESTS_CLOUD_PASSWORD_SMT }}
```

Comment:
were these secrets set up properly? @mzitnik

Reply:
Yep should be fine.